openpgp/ecdh/ecdh.go

```diff
@@ -0,0 +1,226 @@
+package ecdh
+
+import (
+	"bytes"
+	"crypto"
+	"crypto/aes"
+	"crypto/elliptic"
+	"encoding/binary"
+	"errors"
+	"io"
+	"math/big"
+)
+
+type PublicKey struct {
+	elliptic.Curve
+	X, Y *big.Int
+}
+
+type PrivateKey struct {
+	PublicKey
+	X *big.Int
+}
+
+func (e *PublicKey) KDF(S []byte, kdf_params []byte, hash crypto.Hash) []byte {
+	S_len := (e.Curve.Params().P.BitLen() + 7) / 8
```

Comment:
Thank you. Sorry for wasting your time with such trivial issues. I'll try to get a linter running.